src/nni_manager/core/nnimanager.ts

```diff
@@ -44,6 +44,7 @@ class NNIManager implements Manager {
     private readonly: boolean;
 
     private trialJobMetricListener: (metric: TrialJobMetric) => void;
+    public trialJobMessage: Map<string, string | undefined>;
```

Comment:
Already has a `trialJobs` map, the message is a field under trialJobDetail, do not need to add this `trialJobMessage` map again, just use `trialJobs.get(jobId).message`.

Reply:
changed accordingly